endpoints/openrtb2/video_auction.go

```diff
@@ -130,11 +131,11 @@ func (deps *endpointDeps) VideoAuctionEndpoint(w http.ResponseWriter, r *http.Re
 
 	resolvedRequest := requestJson
 	if debugLog.EnableDebug {
-		debugLog.Data = fmt.Sprintf("Request:\n%s", string(requestJson))
```

Comment:
This is starting to get bit complicated and isn't covered by tests. Please pull out into helper/util funcs to test outside of the auction flow.

Reply:
Updated implementation to be cleaner and use a few helper functions. Let me know if there is further cleanup I should do.